hello_test.cc

```diff
@@ -1,10 +1,155 @@
-#include <gtest/gtest.h>
-
-// Demonstrate some basic assertions.
-TEST(HelloTest, BasicAssertions) {
-  // Expect two strings not to be equal.
-  EXPECT_STRNE("hello", "world");
-  // Expect equality.
-  EXPECT_EQ(7 * 6, 42);
-}
-
+#include <gtest/gtest.h>
+#include <vector>
+#include <tuple>
+
+#include "element.h"
+#include "board.h"
+#include "movemap.h"
+
+TEST(Sphere, Constructors) {
+  Sphere sphere;
+
+  for (auto element : ELEMENTLIST) {
+    EXPECT_EQ(sphere.at(element), 0);
+  }
+
+  Sphere sphere2({Element::FIRE});
+  for (auto element : ELEMENTLIST) {
+    EXPECT_EQ(sphere2.at(element), element == Element::FIRE);
+  }

```

Comment:
There should only be one thing tested in each test. Please split this out to a different test.